README.md

```diff
@@ -145,8 +145,8 @@ See also the [full documentation](https://docs.earthly.dev).
 Reference pages
 
 * 📑 [Earthfile reference](https://docs.earthly.dev/earthfile)
-* #️⃣ [Earth command reference](https://docs.earthly.dev/earth-command)
-* ⚙️ [Configuration reference](https://docs.earthly.dev/earth-config)
+* #️⃣ [Earth command reference](https://docs.earthly.dev/earthly-command)
+* ⚙️ [Configuration reference](https://docs.earthly.dev/earthly-config)
```

Comment:
Note that these will require redirects in gitbook. See https://docs.gitbook.com/integrations/github/content-configuration